### ISSUE TITLE
cant select duplicate 'open with' entries

### ISSUE THREAD

**Issue by philipzae** [mate-desktop/caja#246]
i hope this is the correct github for this issue and please let me know if caja doesnt control the 'properties' files dialog.

i right-clicked on a python script file and went to the 'open with' tab and noticed two entries called 'Text Editor', but i'm unable to select the second one, though i dont necessarily have to.

![screenshot-test py properties](https://f.cloud.github.com/assets/4412114/2036830/40ab81a8-8958-11e3-91ab-8a2e6fc61e95.jpg)
##

**Comment by monsta** [comment]
> As stated before, the problem is that the 'Open With' tab has a bug in, as it doesnt allow me to select a second entry which has the same name as another entry.

I can't reproduce it in my LMDE 2 Betsy. I have pluma 1.8.1 and gedit 2.30.5 installed, both have Mint-modified names (set to "Text editor"), and both are selectable in the file properties dialog.

![caja-file-properties](https://cloud.githubusercontent.com/assets/5138986/7706936/ee920c10-fe55-11e4-9637-45f15592bf0c.png)

BTW, I've noticed that gedit is already present on your screenshot (second entry). Are you sure that one of "Text editors" is also gedit? :)

**Comment by monsta** [comment]
Ah, and you used Mint 13 which has MATE only up to 1.6.
If you don't see the issue in Caja 1.8 - either in Mint 17.x or LMDE 2 - then it can be closed as fixed.

**Comment by philipzae** [comment]
Still reproducible with Mint 17.1 Mate with Caja 1.8.2.

Screencast - https://drive.google.com/file/d/0B6qJrVIa0SAld3lfQTl3QkdvckE/view?usp=sharing

**Comment by monsta** [comment]
Wow. Is this still with gedit as the second "Text editor"?
I'm curious about how to reproduce that...

**Comment by philipzae** [comment]
Yes the second "Text editor" is gedit, though it is the first one in the list and the only one i can select.

**Comment by monsta** [comment]
Phew... finally, I can reproduce it with gedit 2.30.5 (from Mint repos) in Mint 17. :)
